FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Configurations/ConfigurationValidators/ValidateUtility.cs

```diff
@@ -0,0 +1,117 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using Microsoft.Health.Fhir.Synapse.Common.Exceptions;
+using Microsoft.Health.Fhir.Synapse.Common.Models.Jobs;
+
+namespace Microsoft.Health.Fhir.Synapse.Common.Configurations.ConfigurationValidators
+{
+    public static class ValidateUtility
```

Comment:
Need we also version these validate utilities?

Reply:
Yes. We may need distinguish this utility by version in the future, which will be determined by the actual difference/changes